composer.json

```diff
@@ -15,12 +15,12 @@
         }
     ],
     "require": {
-        "php": "^7.4 || ^8.0",
-        "silverstripe/cms": "^4",
-        "silverstripe/framework": "^4.11",
-        "silverstripe/admin": "^1",
-        "silverstripe/versioned": "^1",
-        "symfony/yaml": "^3 || ^4"
+        "php": "^8.1",
+        "silverstripe/cms": "^5",
+        "silverstripe/framework": "^5",
```

Comment:
Do we need to require the framework when we already require the cms module? CMS also already requires versioned and admin.

Reply:
You should always explicitly declare your dependencies. If you use framework's classes/APIs you should be declaring it as a dependency. There is no obligation for any of your dependencies (ie: CMS) to always include framework. So if a packaged needs a dep, it should declare it.